backend/api/v2beta1/artifacts.proto

```diff
@@ -0,0 +1,125 @@
+// Copyright 2024 The Kubeflow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option go_package = "github.com/kubeflow/pipelines/backend/api/v2beta1/go_client";
+package kubeflow.pipelines.backend.api.v2beta1;
+
+import "google/api/annotations.proto";
+import "google/api/httpbody.proto";
+import "google/rpc/status.proto";
+import "google/protobuf/timestamp.proto";
+
+service ArtifactService {
+  // Finds all artifacts within the specified namespace.
+  // Namespace field is required. In multi-user mode, the caller
+  // requires RBAC verb "list" on resources "artifacts" for the
+  // specified namespace.
+  rpc ListArtifacts(ListArtifactRequest) returns (ListArtifactResponse) {
+    option (google.api.http) = {
+      get: "/apis/v2beta1/artifacts"
+    };
+  }
+
+  // Finds a specific Artifact by ID.
+  rpc GetArtifact(GetArtifactRequest) returns (Artifact) {
+    option (google.api.http) = {
+      get: "/apis/v2beta1/artifacts/{artifact_id}"
+    };
+  }
+}
+
+message Artifact {
+  // Unique Artifact ID. Generated by MLMD.
+  string artifact_id = 1;
+  // Storage Provider to which this Artifact is located (e.g. S3, Minio, etc.).
+  string storage_provider = 3;
```

Comment:
what happened to the 2nd item for this message (goes direct from 1 to 3)

Reply:
fixed [here](https://github.com/opendatahub-io/data-science-pipelines/commit/e50520648c0f7cc5e43fa4cded9914b9b5ff7c92#diff-e657e17b178a9217bfc1603424e938ba14c37028ec83d7f74bd0a8852bc3340cR112)